pages/api/business/listing/[sku].js

```diff
@@ -0,0 +1,46 @@
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../auth/[...nextauth]";
+import { prisma } from "../../../../db/prismaDB";
+
+export default async function handler(req, res) {
+  const session = await getServerSession(req, res, authOptions);
+  if (!session) {
+    return res.status(401).json({ message: "Unauthorized" });
+  }
+  if (req.method !== "PUT") {
+    return res.status(405).json({ error: "Method not allowed" });
+  }
+  const { sku } = req.query;
+
+  if (!sku) {
+    return res.status(400).json({ message: "sku is missing." });
+  }
+
+  try {
+    const business = await prisma.business.findUnique({
+      where: { email: session.user.email },
+    });
+    console.error(business);
+    if (!business) {
+      return res.status(404).json({ message: "Business record not found" });
+    }
+    await prisma.listing.updateMany({
+      where: {
+        Barcode: sku,
+      },
+      data: { isActive: false },
+    });
+    console.error("working");
+    const listings = await prisma.listing.findMany({
```

Comment:
same here, could it have more than one listing with the same SKU?

Reply:
since Brian has mentioned in the ticket on Asana to use findaMany in the response therefore I used that.